src/include/daos_types.h

```diff
@@ -78,55 +78,6 @@ typedef struct {
 	uint32_t	 cs_chunksize;
 } daos_csum_buf_t;
 
-static inline void daos_csum_set_multiple(daos_csum_buf_t *csum_buf, void *buf,
-					  uint32_t csum_buf_size,
-					  uint16_t csum_size,
-					  uint32_t csum_count,
-					  uint32_t chunksize)
-{
-	csum_buf->cs_csum = buf;
-	csum_buf->cs_len = csum_size;
-	csum_buf->cs_buf_len = csum_buf_size;
-	csum_buf->cs_nr = csum_count;
-	csum_buf->cs_chunksize = chunksize;
-}
-
-static inline bool
-daos_csum_isvalid(const daos_csum_buf_t *csum)
```

Comment:
if we are removing all checksum function from the public API, then we should also remove the types?
daos_csum_buf_t (?) and remove it from daos_iov_t ?

Reply:
OK i recall this now. sorry my bad, you might have had it right before my comment. 
what you did is remove all the csum functions from the API, and internally set the csum in the iod to NULL wherever it is used until we enable checksums, right? i think that's the way to go.
the other one is to keep the function in the API, but do not change them (i.e. keep the old API and build it on top of the new dcb API). the problem with this now, is that people externally using DAOS will have to change their code twice, once when this lands, and another time when this is moved back to private.

Reply:
I'll move the dcb functions back, thanks.